src/server/controllers/participants.js

```diff
@@ -165,3 +166,40 @@ exports.update = async (req, res) => {
     res.send({ error: message });
   }
 };
+
+// Get courses for an individual participant
+exports.getCourses = async (req, res) => {
+  try {
+    const participantId = req.params.id;
+    
+    const participantCourses = await courses.findAll({
+      where: {
+        participant_id: participantId
+      }
+    }); 
+    res.send({ participantCourses });
+  } catch (err) {
+    res.status(500).send({ err });
+  }
+};
+
+// delete an exist course for an individual participant
+exports.deleteCourse = (req, res) => {
+  try {
+    courses
```

Comment:
You could use await here too, as you did with getCourses above:
```
try {
  await courses.destroy({...})
  res.status(200).send({...})
} catch (err) {
 ...
}
```
We really like await at my job 😀

Reply:
I'm not sure if it notifies you since I'm commenting on a merged PR so I'll just @ you, @AsalaKM